### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by RuslanSky23**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,418 @@
...
+            <a href="#apple" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
```
Comment (IvanFesenko) [inline src/index.html:34]: don`t forget to add empty line between sibling tags

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,418 @@
...
+            class="card__images"
+          >
+
+          <div class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__code">
+            Product code: 195434
+          </div>
```
Comment (IvanFesenko) [inline src/index.html:88]: Do not use div for text content - use semantic tags instead
h2-h5 for heading, p for other text
Fix all similar cases

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,418 @@
...
+
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__reviews">
+              Reviews: 5
+            </div>
```
Comment (IvanFesenko) [inline src/index.html:98]: This text should not belong stars block. Move it out

File: src/styles/blocks/header.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    top: 10px;
+    left: 50px;
+  }
```
Comment (IvanFesenko) [inline src/styles/blocks/header.scss:15]: Do not use absolute here

File: src/styles/blocks/stars.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
+@import "../utils/variables";
+
+.stars {
+  display: flex;
+  flex-direction: row;
```
Comment (IvanFesenko) [inline src/styles/blocks/stars.scss:5]: flex-direction: row; is default value - redudant

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
hover effect should be on eash menu item, not only 4th
https://github.com/mate-academy/layout_catalog/assets/61904995/286e4c10-637f-4585-aa8b-58399a875343

It should be always 4 card in a row
2023-05-17_16h53_43

File: readme.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://RuslanSky23.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://RuslanSky23.github.io/layout_catalog/report/html_report/)
```
Comment (vlad-demchuk) [inline readme.md:4]: add links to PR description

File: src/index.html (hunk 1 above)
Comment (vlad-demchuk) [reply src/index.html:34]: not fixed
check examples in the checklist
image

File: src/index.html (hunk 2 above)
Comment (vlad-demchuk) [reply src/index.html:88]: https://github.com/mate-academy/layout_catalog/pull/3145/files#r1196550136
not fixed

File: src/index.html (hunk 3 above)
Comment (vlad-demchuk) [reply src/index.html:98]: the same
https://github.com/mate-academy/layout_catalog/pull/3145/files#r1196551121

File: src/styles/blocks/header.scss [hunk 7]
```diff
@@ -0,0 +1,68 @@
+@import "../utils/variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    position:fixed;
```
Comment (vlad-demchuk) [inline src/styles/blocks/header.scss:10]: it should be neither fixed nor absolute

just use margins to place it correctly

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

**Comment by RuslanSky23** [comment]
- [DEMO LINK](https://RuslanSky23.github.io/layout_catalog/)
- [TEST REPORT LINK](https://RuslanSky23.github.io/layout_catalog/report/html_report/)

**Pull request closed without merging**